napari/_qt/_tests/test_exception_handler.py

```diff
@@ -15,3 +18,6 @@ def test_exception_handler(qtbot, caplog):
     assert record.levelname == 'ERROR'
     assert 'Unhandled exception' in record.message
     assert 'ValueError: whoops' in record.message
+    with pytest.raises(SystemExit):
```

Comment:
This is great! But generally I'd prefer a separate test, rather than a single test testing every single behaviour.

Reply:
this has now been done